network/parallelism.go

```diff
@@ -12,33 +12,27 @@ import (
 
 // parallelism drives the behavior on determining what the desired number of connections should be.
 type parallelism struct {
-	// mut covers all items here
-	mut      sync.RWMutex
-	cfg      types.ParallelismConfig
-	statshub types.StatsHub
-	ctx      context.Context
-	// networkErrors is any 4xx,5xx.
-	// network* holds the time any success or error occurs.
-	// This is used for the lookback so we can drop any that are older than our `cfg.Lookback`.
-	networkErrors    []time.Time
-	networkSuccesses []time.Time
-
-	timestampDriftSeconds int64
-	currentDesired        uint
-	out                   *types.Mailbox[uint]
-	stop                  chan struct{}
-	// previous is the number of previous desired instances. This is to prevent flapping.
-	previous                   []previousDesired
-	networkRelease             types.NotificationRelease
+	ctx                        context.Context
+	l                          log.Logger
+	statshub                   types.StatsHub
 	serializerRelease          types.NotificationRelease
+	out                        *types.Mailbox[uint]
+	stop                       chan struct{}
+	networkRelease             types.NotificationRelease
+	networkErrors              []time.Time
+	networkSuccesses           []time.Time
+	previous                   []previousDesired
+	cfg                        types.ParallelismConfig
+	timestampDriftSeconds      int64
+	currentDesired             uint
 	timestampNetworkSeconds    int64
 	timestampSerializerSeconds int64
-	l                          log.Logger
+	mut                        sync.RWMutex
```

Comment:
Technically this one is fine, but we often have a pattern of putting the mux over the data it is protecting so eh. Ship it if it makes it more consistent and gets rid of linting.

Reply:
I'm fine with disabling the fieldalignment linter by default, I think it's interesting but definitely isn't important for most structs.